components/Questionnaire/index.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+  <div id="registration" class="registration-questionnaire">
+    <div class="registration--position">
+      <img class="registration__image" src="~/assets/images/FooterBanner.svg" alt="footer banner">
+      <div class="registration__questionnaire">
+        <div v-if="!showModalLevelDesa">
+          <QuestionnaireOne v-show="showLevelOne" @onClickLevel="validationQuestionnaireOne" @onSubmit="onNextLevelOne" />
+          <QuestionnaireTwo v-show="showLevelTwo" @onClickLevel="validationQuestionnaireTwo" @onPrev="onPrev" @onSubmit="onNextLevelTwo" />
+          <QuestionnaireThree v-show="showLevelThree" />
+        </div>
+        <ModalQuestionnaire v-else :chosen-level="params.level" :village-types="villages" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import {
+  villages
+} from '@/constants/questionnaire.js'
+export default {
+  data () {
+    return {
+      params: {
+        id: '32.09.21.2001', // @todo: remove this in next feature (id desa search)
+        level: 1,
+        properties: {
+          fasilitas_desa: {
+            akses_kendaraan: {
```

Comment:
There is a lot of unused data, will it be used in the future? 🤔

Reply:
Yes, this data inside params will be used in the future for the next PR @agunghide

Reply:
oh, i see. Thanks for the answer